torch/onnx/symbolic_helper.py

```diff
@@ -800,6 +800,35 @@ def _interpolate_get_scales_and_mode(g, input, size, scale_factor, mode, align_c
     return scale_factor, mode
 
 
+def _argmin_argmax_helper(g, input, dim, keepdim, op_name):
```

Comment:
Could you help annotate the type here? I would add `@parse_args(...)` to `argmax` etc. so that this function can take python types like `int`

Reply:
I'm wondering can `@parse_args` handle `dim=None`?

Reply:
Probably not at this point but we can add support. For now just parse as "v"?